src/apps/socket/unix.lua

```diff
@@ -154,7 +154,6 @@ function selftest ()
             if l == nil then return end
             while not link.empty(l) do
                local p = link.receive(l)
-               print(name..': ', p.length, ffi.string(p.data, p.length))
```

Comment:
Instead of removing this line, perhaps is better to assert a packet with data was received.